esmvaltool/diag_scripts/monitor/multi_datasets.py

```diff
@@ -292,6 +298,47 @@
     coordinates. Can be adjusted to avoid overlap with the figure. Only
     relevant if ``show_stats: true``.
 
+
+Configuration options for plot type ``1d_profile``
+--------------------------------------------------
+plot_bias: string, optional
+    Optional switch to choose whether difference plot to reference should
+    be applied. And if plot_bias is not False or empty, it defines if
+    difference is 'absolute' or 'relative'.
+legend_kwargs: dict, optional
+    Optional keyword arguments for :func:`matplotlib.pyplot.legend`. Use
+    ``legend_kwargs: false`` to not show legends.
+gridline_kwargs: dict, optional
+    Optional keyword arguments for grid lines. By default, ``color: lightgrey,
+    alpha: 0.5`` are used. Use ``gridline_kwargs: false`` to not show grid
+    lines.
+aspect_ratio: float, optional
+    Optional set the aspect ratio of the one-dimensional plot. Default is 3/2,
+    which results in a slender upright plot.
+y_log: bool, optional
+    Optional set a logarithmic scale for the vertical axis
+x_log: bool, optional
+    Optional set a logarithmic scale for the x-axis
```

Comment:
```suggestion
log_x: bool, optional (default: False)
    Use logarithmic X-axis.
log_y: bool, optional (default: True)
    Use logarithmic Y-axis.
```

This is called `log_x` and `log_y` in the code